packages/core/content-manager/admin/src/history/components/VersionInputRenderer.tsx

```diff
@@ -218,6 +218,43 @@ type VersionInputRendererProps = DistributiveOmit<EditFieldLayout, 'size'> & {
   shouldIgnoreRBAC?: boolean;
 };
 
+/**
+ * Checks if the i18n plugin added a label action to the field and modifies it
+ * to adapt the wording for the history page.
+ */
+const getLabelAction = (labelAction: VersionInputRendererProps['labelAction']) => {
+  if (!React.isValidElement(labelAction)) {
+    return labelAction;
+  }
+
+  const labelActionTitleId = labelAction.props.title.id;
```

Comment:
```suggestion
	// TODO: find a better way to do this rather than access internals
  const labelActionTitleId = labelAction.props.title.id;
```

Reply:
just feel like it's one step away from being perfect, but this would take ages to figure out.